### PR TITLE
Change Volume File Ownership Jobs for Security Contexts

### PR DESCRIPTION
**Pull request by mphammer**
synopsyctl creates jobs to change ownership in Volumes when Security Contexts are enabled/changed

### CODE REVIEW

**Comment by operator-r2d2** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/blackducksoftware/synopsys-operator/pull/1179#" title="Author self-approved">mphammer</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=blackducksoftware%2Fsynopsys-operator).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/blackducksoftware/synopsys-operator/blob/release-2019.12.x/OWNERS)~~ [mphammer]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: pkg/util/security_context.go [hunk 1]
```diff
@@ -35,7 +35,8 @@ func SetSecurityContextInPodConfig(podConfig *PodConfig, securityContext *Securi
 		podConfig.RunAsGroup = securityContext.RunAsGroup
 
 		if !isOpenshift {
-			podConfig.FSGID = securityContext.FsGroup
+			//podConfig.FSGID = securityContext.FsGroup
+			podConfig.FSGID = IntToInt64(0)
```
Comment (mphammer) [inline pkg/util/security_context.go]: I will explicitly prevent the user from setting this in another PR

**Comment by operator-r2d2** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/blackducksoftware/synopsys-operator/pull/1179#issuecomment-574302229" title="Approved">jdartigalongue</a>*, *<a href="https://github.com/blackducksoftware/synopsys-operator/pull/1179#" title="Author self-approved">mphammer</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=blackducksoftware%2Fsynopsys-operator).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/blackducksoftware/synopsys-operator/blob/release-2019.12.x/OWNERS)~~ [jdartigalongue,mphammer]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by yashbhutwala** [comment]
```
1. old < 12.0 && new >= 12.0
  1.1 old was edited to have security context in CR (with all fields set ideally, specifically runAsUser is checked)
  1.2 old was not edited to have security context in CR (normal upgrade; no security context change)
2. security context flag && new >= 12.0
```

/lgtm

**Comment by operator-r2d2** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/blackducksoftware/synopsys-operator/pull/1179#issuecomment-574302229" title="Approved">jdartigalongue</a>*, *<a href="https://github.com/blackducksoftware/synopsys-operator/pull/1179#" title="Author self-approved">mphammer</a>*, *<a href="https://github.com/blackducksoftware/synopsys-operator/pull/1179#issuecomment-574313739" title="LGTM">yashbhutwala</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=blackducksoftware%2Fsynopsys-operator).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/blackducksoftware/synopsys-operator/blob/release-2019.12.x/OWNERS)~~ [jdartigalongue,mphammer,yashbhutwala]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by operator-r2d2** [comment]
@mphammer: new pull request created: #1180

<details>

In response to [this](https://github.com/blackducksoftware/synopsys-operator/pull/1179#issuecomment-574325777):

>/cherrypick master


Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

**Pull request merged**